http-client/src/lib.rs

```diff
@@ -143,33 +138,40 @@ impl CapabilityProvider for HttpClientProvider {
     }
 
     /// Handle all calls from actors.
-    fn handle_call(&self, actor: &str, op: &str, msg: &[u8]) -> Result<Vec<u8>, Box<dyn Error>> {
+    fn handle_call(
+        &self,
+        actor: &str,
+        op: &str,
+        msg: &[u8],
+    ) -> Result<Vec<u8>, Box<dyn Error + Send + Sync>> {
         match op {
             OP_BIND_ACTOR if actor == SYSTEM_ACTOR => self.configure(deserialize(msg)?),
             OP_REMOVE_ACTOR if actor == SYSTEM_ACTOR => self.deconfigure(deserialize(msg)?),
             OP_PERFORM_REQUEST => self.request(actor, deserialize(msg)?),
-            OP_GET_CAPABILITY_DESCRIPTOR => self.get_descriptor(),
             _ => Err(format!("Unknown operation: {}", op).into()),
         }
     }
+
+    fn stop(&self) {
+        todo!()
```

Comment:
This will cause a runtime exception when it hits. We should remove and replace with a no-op.